client/app/cloud/project/compute/infrastructure/list/cloud-project-compute-infrastructure-list.html

```diff
@@ -3,89 +3,87 @@
+                                data-ng-disabled="ComputeInfrastructureDiagramCtrl.loaders.vRack"
+                                data-toggle="dropdown"
+                                write-right-required>
+                            <span data-ng-bind=":: 'common_actions' | translate"></span>
+                            <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
+                        </button>
+                        <ul class="dropdown-menu mt-0" role="menu">
+                            <li role="menuitem">
+                                <a class="btn fs14 text-left" ui-sref="iaas.pci-project.compute.infrastructure.vm-add()"
+                                   at-internet-click="{ name : 'cloud-project::cloud-project-compute::cloud-project-compute-infrastructure-add-virtual-machine' }"
+                                   data-ng-bind=":: 'cpci_add_server' | translate">
+                                </a>
+                            </li>
+                            <li>
+                                <button type="button" class="btn fs14"
+                                        at-internet-click="{ name : 'cloud-project::cloud-project-compute::cloud-project-compute-infrastructure-add-volume' }"
+                                        data-ng-bind=":: 'cpci_volume_add' | translate"
+                                        data-ng-click="ComputeInfrastructureListCtrl.InfrastructureService.addVolume()">
+                                </button>
+                            </li>
+                            <li class="divider" role="presentation"></li>
+                            <li role="menuitem">
+                                <button type="button"
+                                        class="btn fs14"
+                                        data-ui-sref="iaas.pci-project.compute.infrastructure.iac-view"
+                                        data-at-internet-click="{
+                                            name: 'cloud-project::cloud-project-compute::cloud-project-compute-infrastructure-add-view'
+                                        }"
+                                        data-ng-bind=":: 'cpci_view_iac' | translate ">
+                                </button>
+                            </li>
+                            <li class="divider" role="presentation"></li>
+                            <li>
+                                <button type="button" class="btn fs14"
+                                        at-internet-click="{ name : 'cloud-project::cloud-project-compute::cloud-project-compute-infrastructure-buy-ip' }"
+                                        data-ng-bind=":: 'cpci_buy_ip_failover' | translate"
+                                        data-ng-click="ComputeInfrastructureListCtrl.InfrastructureService.buyIpFailOver()">
+                                </button>
+                            </li>
+                            <li class="divider"
```

Comment:
```suggestion
                            <li class="divider" role="presentation"
```

Reply:
@FredericEspiau your suggestion is quite interesting but in that context why you didn't suggest `role="separator"`?

Here I'm referring to the [Bootstrap documentation](https://getbootstrap.com/docs/3.3/components/#navbar) for instance.